docs/asb/README.md

```diff
@@ -0,0 +1,134 @@
+# XMR to BTC Atomic Swap - Automated Swap Backend (ASB)
+
+## Quick Start ASB
+
+- Ensure that you have the Monero Wallet RPC running with `wallet-dir` and `disable-rpc-login`:
+  - `monero-wallet-rpc --stagenet --daemon-host STAGENET-NODE-URL --rpc-bind-port STAGENET-NODE-PORT --disable-rpc-login --wallet-dir PATH/TO/WALLET/DIR`
+- Run the ASB in terminal: `./asb start`
+- Follow the setup wizard in the terminal
```

Comment:
Using 1., 2. and 3. might be better here because the order is actually important.